package.json

```diff
@@ -20,27 +20,28 @@
     "amqp-match": "^0.0.0",
     "apollo-server-express": "^1.3.4",
     "body-parser-graphql": "^1.0.0",
+    "compress": "^0.99.0",
```

Comment:
Where can I find the `README` for this package. I found https://www.npmjs.com/package/compress but it has nothing on it.